src/Orchard/Tasks/Locking/Services/IDistributedLock.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Orchard.Tasks.Locking.Services {
+    /// <summary>
+    /// Represents a distributed lock.
+    /// </summary>
+    public interface IDistributedLock : IDisposable {
```

Comment:
I agree, there's no point in keeping the abstraction anymore after this changeset. Removing.